plasma_framework/contracts/src/exits/utils/ExitBounty.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity 0.5.11;
+
+library ExitBounty {
+
+    /**
+     * @notice Returns the Process Exit Bounty size for standard exits
+     * @dev See https://github.com/omgnetwork/plasma-contracts/issues/658 for discussion about size
+     * 107000 is the approx gas usage for calling processExit()
+     */
+    function processStandardExitBountySize() internal view returns (uint256) {
+        return 107000 * tx.gasprice;
```

Comment:
- As for this implementation and design, it might be better to pass in gas price as args instead of using `tx.gasprice` for easier testing. I was actually surprise you can call a view function and pass in the gas price in the call to the Wrapper contract 😅 as it is not consuming any gas.
- However, if we are adding upgrade mechanism, I think it will be easier to be the same as what we do for bond. So we don't use `tx.gasprice` to do the calculation but hardcode one price there.
- Minor: slightly concern the abstract might break a bit as ExitBounty quite likely would need a storage when introducing upgrade mechanism. But would leave that as out of scope just a warning this seems to break in the future.

Reply:
passing gas price in args will indeed be better 👍 . 
For the upgrade mechanism, i was thinking more in lines of updating the multiplier ["K"](https://github.com/omgnetwork/plasma-contracts/issues/658#issue-659410568) instead of a new value for the size, so we would not need to update the bounty size frequently and can accommodate frequent gas changes? What do you think?